lib/plugins/bullMqMetricsPlugin.spec.ts

```diff
@@ -138,4 +142,37 @@ describe('bullMqMetricsPlugin', () => {
       'bullmq_jobs_finished_duration_count{status="completed",queue="test_job"} 1',
     )
   })
+
+  it('works with multiple redis clients', async () => {
+    app = await initAppWithBullMqMetrics({
+      redisClient: [redis, redis],
+      collectionOptions: {
+        type: 'manual',
+      },
+    })
+
+    // exec collect to start listening for failed and completed events
+    await app.bullMqMetrics.collect()
+
+    const responseBefore = await getMetrics()
+    expect(responseBefore.result.body).not.toContain(
+      'bullmq_jobs_finished_duration_count{status="completed",queue="test_job"} 1',
```

Comment:
It was supposed to be 2 here to check if the value we expect wasn't there before. But it will be even better to check that the metric does not exist at all at start, instead of checking for specific value, I will adjust

Reply:
Updated ✅